os_net_config/impl_nmstate.py

```diff
@@ -50,6 +50,27 @@
 # Import the raw NetConfig object so we can call its methods
 netconfig = os_net_config.NetConfig()
 
+_VF_BIND_DRV_SCRIPT = '''
+dpdk_vfs="{dpdk_vfs}"
+linux_vfs="{linux_vfs}"
+set -x
+
+for vfid in $dpdk_vfs $linux_vfs; do
+    vf_pci_id=$(basename $(readlink /sys/class/net/$1/device/virtfn$vfid)) &&
```

Comment:
Just to be safe, can you split this into two commands?

```bash
vf_pci_id=$(readlink "/sys/class/net/$1/device/virtfn$vfid") &&
vf_pci_id=$(basename "$vf_pci_id") &&
```

That way, if `readlink` fails, it will abort immediately.

Reply:
done